app/views/trn_requests/show.html.erb

```diff
@@ -7,6 +7,11 @@
     <p class="govuk-body">We will send your TRN to <%= @trn_request.email %> if we find one matching your details.</p>
 
     <% rows = [
+        { 
+          key: { text: 'Your Name' }, 
+          value: { text: @trn_request.name }, 
+          actions: [{ href: name_path, visually_hidden_text: 'name' }]
+        },
```

Comment:
Could we also play back `Previous name` here as a separate section?

Reply:
Done.

<img width="662" alt="Screen Shot 2022-03-11 at 8 59 04 am" src="https://user-images.githubusercontent.com/3126/157835554-7609b90a-c0dc-41c6-a729-859984865eee.png">

Reply:
Nice, could they be combined into just `Previous name`?

Reply:
Sure. So, if only one previous name field is present, combine it with the other existing name to get a full name?

eg. Joan Smith enters a previous last name of Doe, so we display Joan Doe.

Reply:
I like this approach